packages/docs/community/release-process/smart-contracts.md

```diff
@@ -37,57 +37,71 @@ When a release is getting proposed on a Baklava/Alfajores/Mainnet, the commit th
 After a completed release process, the release branch should be merged into `master` with a merge commit (instead of the usual squash merge strategy).
 
 
-## Build and Release Process
+## Release Process
 
-Using the following scripts, new contracts can be built and deployed alongside a corresponding on-chain governance proposal. Run the following scripts from the branch with the new release contracts:
+Using the following procedure, a new contract release can be built, deployed, and proposed for upgrade automatically.
 
+### Verify Current Deployment
+
+Compile the contracts in <code>$RELEASE_BRANCH</code> branch and verify that the deployed bytecode on <code>$NETWORK</code> matches compiled bytecode.
```

Comment:
Can we reference the specific tags from above? I.e. `celo-core-contracts-v${N-1}.(baklava | alfajores | mainnet)`

Reply:
And maybe just include the command to checkout and compile?

Reply:
this does checkout and compile :)